projects/browser-phase-1.md

```diff
@@ -0,0 +1,161 @@
+
+OpenTelemetry API review:
+* Package size
+* Dependencies
+* Other browser requirements that differ from NodeJS
+
+Data Modelling:
+* Sessions
+* Resources / Entities
+* Navigation
+* Anonymous user ids (Manager)
+
+The SIG will also evaluate how best to maintain browser-specific packages, in terms of teams, repos, and other GitHub code management policies.
+
+### Support
+
+We will also decide our compatibility story.
+* Which versions of which browsers do we plan to officially support?
+* Will we aim support electron at this time?
+
+### Step 2: Core Instrumentation
+
+Once we are clear on what API packages we plan on using, we plan to implement an initial set of instrumentation packages for high priority browser libraries. The goal of this step is to provide enough of an initial ecosystem that end users and vendors can validate the decisions made in Step 1, along with the Semantic Conventions, so that we can stabilize these decisions based on real world feedback.
+
+During this phase, the SIG will not be focused on implementing a browser-optimized SDK. Instead, we will continue to use the existing OTel JS SDK as our reference implementation. Third party SDKs may also choose to bind to the OTel Browser API at this time.
+
+In some cases, browser instrumentation already exists but may be different from the new instrumentation we want to provide. This instrumentation should be treated as "de facto stable" and should not experience breaking changes until we are ready to issue a stable v1.0. New unstable versions of this instrumentation should be managed in a way that allow the SIG to move quickly without destabilizing any instrumentation currently in production.
+
+Instrumentation for the browser runtime:
+* Page load/unload
+* User events (clicks, etc)
+* Resource timing
+* Errors
+* Web vitals
+* Long tasks
+
+Example list of library instrumentation:
+* Fetch / XHR Requests
+* Websockets
+* React
+* Next
+* Svelte
+* Angular
+* Vue.js
+
+### Next Steps
+
+Once we are satisfied that we have achieved the above goals, this SIG will review the remaining work and available resources and choose a new project for the SIG to work on. 
+
+Options include:
+* Continue implementing instrumentation packages.
+* Implement a browser-specific SDK.
+* Implement a more efficient client protocol.
+* Implement a public gateway for client protocols.
+* TBD
+
+## Staffing / Help Wanted
+
+### Required Staffing
+
+**Project Lead:** Ted Young
+
+**GC Liaison:** Daniel Gomez Blanco
+
+**Sponsors:**
+* Ted Young
+* SPONSOR NEEDED
+
+**Implementation Engineers:**
+* Martin Kuba
+* Purvi Kanal
+* Nev Wylie (Microsoft)
+* Karlie L (Microsoft)
+
+	
+**NodeJS Maintainers and Approvers:**
+
+When bootstrapping our browser work, we want to make sure that the existing OpenTelemetry Javascript community is involved. Existing OTel JS maintainers and approvers should participate in this project to help ensure that it is successful.
```

Comment:
Whats the path to add more folks involved? Is it similar to the existing process?

Reply:
Is there specific work you are interested in taking on?

Reply:
@vigneshshanmugam Thank you! Would you be willing to be listed as an implementation engineer? Meaning that you will make some time to write code in addition to reviewing designs?

Reply:
Missed this comment, yes happy to get involved in both the implementation and design. Thanks for the poke.